src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_is_moved_to_error_queue_using_dtc.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using System.Transactions;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_message_is_moved_to_error_queue_using_dtc : NServiceBusAcceptanceTest
```

Comment:
@timbussmann @tmasternak Should we handle this missing test case as a separate GitHub issue? I'd love to move towards closing this PR (and parent issue) without creating more work within the issue at hand, if possible.